console/src/config.rs

```diff
@@ -28,6 +29,11 @@ pub struct Config {
 
     #[clap(flatten)]
     pub(crate) view_options: ViewOptions,
+
+    /// The amount of time resource that have been dropped
+    /// will be displayed in the console.
```

Comment:
how about something like this?
```suggestion
    /// How long to continue displaying completed tasks and dropped resources
    /// after they have been closed.
```

Reply:
we may want to also add a note about how durations can be specified so the user knows what kinds of input will be accepted. this, of course, will depend on whether we continue using our homemade duration parsing code or switch to the `humantime` crate...